src/main/java/io/confluent/kafkarest/Errors.java

```diff
@@ -93,7 +93,16 @@ public static RestException consumerAlreadyExistsException() {
                              CONSUMER_ALREADY_EXISTS_ERROR_CODE);
   }
 
+  public final static String ILLEGAL_STATE_MESSAGE = "Illegal state: ";
+  public final static int ILLEGAL_STATE_ERROR_CODE = 40903;
 
+  public static RestException illegalStateException(Throwable t) {
```

Comment:
Ah, I missed this when commenting about `IllegalStateException` above -- 409 makes perfect sense.